audius-cli

```diff
@@ -542,7 +542,7 @@ def get_config(ctx, service):
 
 
 @cli.command()
-@click.option("--unset", is_flag=True)
+@click.option("--unset", is_flag=True) # No longer used. Use `audius-cli unset-tag` instead.
```

Comment:
are we keeping this for backwards compatibility?

Reply:
compatibility in a way, yeah, to alert users that it doesn't work and that they should use `audius-cli unset-tag` instead

Reply:
"users" meaning SPs or devs who try to unset their tag in this way on a stage or foundation node